src/conductor/op-conductor/launcher.star

```diff
@@ -93,26 +107,23 @@ def get_service_config(
             params.service_name, consensus_port.number
         ),
         "OP_CONDUCTOR_CONSENSUS_PORT": str(consensus_port.number),
-        "OP_CONDUCTOR_EXECUTION_RPC": sidecar_context.rpc_http_url
-        if sidecar_context
-        else _net.service_url(
-            el_params.service_name, el_params.ports[_net.RPC_PORT_NAME]
+        "OP_CONDUCTOR_EXECUTION_RPC": (
+            sidecar_context.rpc_http_url
+            if sidecar_context
+            else _net.service_url(
+                el_params.service_name, el_params.ports[_net.RPC_PORT_NAME]
+            )
         ),
         "OP_CONDUCTOR_NODE_RPC": _net.service_url(
             cl_params.service_name, cl_params.ports[_net.RPC_PORT_NAME]
         ),
         "OP_CONDUCTOR_ROLLUP_BOOST_ENABLED": "true" if sidecar_context else "false",
-        # This might also become a parameter
-        "OP_CONDUCTOR_HEALTHCHECK_INTERVAL": str(_CONDUCTOR_HEALTH_CHECK_INTERVAL),
-        # This might also become a parameter
+        "OP_CONDUCTOR_HEALTHCHECK_INTERVAL": str(params.healthcheck_interval),
         "OP_CONDUCTOR_HEALTHCHECK_MIN_PEER_COUNT": str(
-            _CONDUCTOR_HEALTH_CHECK_MIN_PEER_COUNT
+            params.healthcheck_min_peer_count
         ),
-        # docs recommend a 2-3x multiple of your network block time to account for temporary performance issues
-        #
-        # TODO This might be later added as a multiplier parameter if needed
         "OP_CONDUCTOR_HEALTHCHECK_UNSAFE_INTERVAL": str(
-            network_params.seconds_per_slot * 2 + 1
+            params.healthcheck_unsafe_interval
```

Comment:
This will defintely make your conductors go weird - the original value was `network_params.seconds_per_slot * 2 + 1 = 2 * 2 + 1 = 5` and it was dynamically dependent on the block time, the new value is hardcoded to `600`.

If we want to move it to the params so it can be modified, I would keep the default value the same (i.e. computed based on the blocktime, not hardcoded)